### PR TITLE
Add sym / cmp for I²C ADC / DAC

### PR DESCRIPTION
**Pull request by dbrgn**
img

### CODE REVIEW

**Comment by dbrgn** [comment]
@ubruhin updated pin positions as requested:

img

**Comment by dbrgn** [comment]
Updated:

- New shape
- Renamed VDD/VSS to V+/V-
- Added "single-ended" to name

I also added a matching symbol for a DAC.

adc

dac

**Comment by dbrgn** [comment]
Rebased again:

- Renamed symbols from "I²C" to "5-Pin" (more generic)
- Added "single-channel" to description
- Added cmp for I²C ADC/DAC

File: cmp/050f753f-1323-43ae-ad72-3f23d7567393/component.lp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ (name "DAC I²C Single-Ended")
+ (description "A generic single-ended single-channel I²C digital-to-analog converter.")
+ (keywords "dac,i2c,single-ended,single-channel")
+ (author "Danilo B.")
+ (version "0.1")
+ (created 2018-12-02T14:10:04Z)
+ (deprecated false)
+ (category f87a2a14-4dd2-49f0-b93d-0c78ac50cf53)
+ (schematic_only false)
+ (default_value "{{ PARTNUMBER or DEVICE or COMPONENT }}")
```
Comment (ubruhin) [inline cmp/050f753f-1323-43ae-ad72-3f23d7567393/component.lp]: It's not a (manufacturer-)specific component, thus it doesn't make sense to see its name in the schematics. Also remove spaces for consistency.
```suggestion
 (default_value "{{PARTNUMBER or DEVICE}}")
```

File: cmp/2ba355ec-4ec2-4a28-8163-270662fd8543/component.lp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ (name "ADC I²C Single-Ended")
+ (description "A generic single-ended single-channel I²C analog-to-digital converter.")
+ (keywords "adc,i2c,single-ended,single-channel")
+ (author "Danilo B.")
+ (version "0.1")
+ (created 2018-12-02T14:08:47Z)
+ (deprecated false)
+ (category a621d579-8715-4edf-816e-5715f97cd2d7)
+ (schematic_only false)
+ (default_value "{{ PARTNUMBER or DEVICE or COMPONENT }}")
```
Comment (ubruhin) [inline cmp/2ba355ec-4ec2-4a28-8163-270662fd8543/component.lp]: Same here.
```suggestion
 (default_value "{{PARTNUMBER or DEVICE}}")
```

File: sym/db769dbd-5ecb-43cf-8259-e2066425f412/symbol.lp [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  (width 0.254) (fill false) (grab_area true)
+  (vertex (position -10.16 0.0) (angle 0.0))
+  (vertex (position -2.54 5.08) (angle 0.0))
+  (vertex (position 10.16 5.08) (angle 0.0))
+  (vertex (position 10.16 -5.08) (angle 0.0))
+  (vertex (position -2.54 -5.08) (angle 0.0))
+  (vertex (position -10.16 0.0) (angle 0.0))
+ )
+ (text f26e51c7-09b7-42dc-aea1-4bce0bade489 (layer sym_names) (value "{{NAME}}")
+  (align left bottom) (height 3.0) (position 2.54 5.08) (rotation 0.0)
```
Comment (ubruhin) [inline sym/db769dbd-5ecb-43cf-8259-e2066425f412/symbol.lp]: Same here, but right-aligned:
```suggestion
  (align right bottom) (height 3.0) (position 10.16 5.08) (rotation 0.0)
```

File: sym/d2c0e7b4-b53c-4325-8c61-eff6d7faf7c3/symbol.lp [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  (width 0.254) (fill false) (grab_area true)
+  (vertex (position 10.16 0.0) (angle 0.0))
+  (vertex (position 2.54 5.08) (angle 0.0))
+  (vertex (position -10.16 5.08) (angle 0.0))
+  (vertex (position -10.16 -5.08) (angle 0.0))
+  (vertex (position 2.54 -5.08) (angle 0.0))
+  (vertex (position 10.16 0.0) (angle 0.0))
+ )
+ (text bc465e7b-a865-470b-9a23-4febc3886c5b (layer sym_names) (value "{{NAME}}")
+  (align right bottom) (height 3.0) (position -2.54 5.08) (rotation 0.0)
```
Comment (ubruhin) [inline sym/d2c0e7b4-b53c-4325-8c61-eff6d7faf7c3/symbol.lp]: Left-aligned looks much better (component names are usually very short, so it's unlikely that it ever overlaps with the V+ pin):
```suggestion
  (align left bottom) (height 3.0) (position -10.16 5.08) (rotation 0.0)
```

File: cmp/050f753f-1323-43ae-ad72-3f23d7567393/component.lp (hunk 1 above)
Comment (dbrgn) [reply cmp/050f753f-1323-43ae-ad72-3f23d7567393/component.lp]: That was actually copied from [the docs](https://developers.librepcb.org/de/df4/doc_attributes_system.html), so you should probably remove the inner spaces there too :slightly_smiling_face:

Comment (ubruhin) [reply cmp/050f753f-1323-43ae-ad72-3f23d7567393/component.lp]: Ah true, even within the docs it's not consistent :(

**Comment by dbrgn** [comment]
Fixed and rebased.

**Pull request merged**